xs2a-adapter-rest-impl/src/main/java/de/adorsys/xs2a/adapter/config/SwaggerConfig.java

```diff
@@ -17,9 +17,19 @@
     @Primary
     public SwaggerResourcesProvider swaggerResourcesProvider() {
         return () -> {
-            SwaggerResource swaggerResource = new SwaggerResource();
-            swaggerResource.setLocation("/openapi.json");
```

Comment:
What about xs2a api?

Reply:
As we discussed with Mark on the daily meeting, we do not want to expose it because it will be replaced by psd2 api

Reply:
added xs2a api